v2/manifests/core-ons/zebedee.yml

```diff
@@ -48,6 +48,7 @@ services:
       OTEL_JAVAAGENT_ENABLED:         ${OTEL_JAVAAGENT_ENABLED:-"false"}
       ELASTIC_SEARCH_CLUSTER:         "cluster"
       ELASTIC_SEARCH_SERVER:          "elasticsearch"
+      brian_url:                      "http://project-brian:8083"
```

Comment:
Could you add this to the Zebedee readme as well?

Reply:
Done :)